pkg/subnet/local_test.go

```diff
@@ -164,7 +169,7 @@ func TestGetLatestAvagoVersion(t *testing.T) {
 func getTestClientFunc() (client.Client, error) {
 	c := &mocks.Client{}
 	fakeStartResponse := &rpcpb.StartResponse{}
-	c.On("Start", mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(fakeStartResponse, nil)
+	c.On("Start", mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(fakeStartResponse, nil)
```

Comment:
Huh, why is this needed?

Reply:
A test was failing, not sure why it didn't break the pipeline on main. This was required for it to pass.

Reply:
Looks like I added a new argument to start, h/t @holisticode